dali/test/python/test_backend_impl.py

```diff
@@ -236,7 +236,7 @@ def dtype_pipeline(np_type, placeholder_dali_type):
 
 
 def test_dtype_converion():
-    dali_types = [types._DALIDataType.INT8, types._DALIDataType.UINT64, types._DALIDataType.FLOAT16]
```

Comment:
This was intentional, please revert.

We should probably add a comment why. It already is explained a bit in backend_impl though - the _DALIDataType is the type inheriting from DALIDataType, extended with `__call__` that supports returning the numpy/array_interface type string.

We want to make sure, that it works as a `dtype` parameter to operators without any issues, as someone may use it that way.

Reply:
Fixed. nvidia.dali.backend_impl.types collided with nvidia.dali.types